include/abb_librws/rws_retcode.h

```diff
@@ -0,0 +1,27 @@
+#ifndef RWS_RETCODE_H
+#define RWS_RETCODE_H
+
+#include <string>
+
+namespace abb
+{
+namespace rws
+{
+/**
+ * \brief Namespace for response code domains.
+ */
+namespace retcode
+{
+    struct RetcodeInfo
+    {
+        std::string name;
+        std::string code;
```

Comment:
It thought about it, but I didn't want to assume it's always an int. 
Just by looking at just the values of [possible return codes](https://developercenter.robotstudio.com/api/rwsApi/common_return_codes.html) there are just ints, but the progress service is undocumented AFAIK, so I left it as string.
But now as I think about it, it should be an int and potential not-an-int values should be handled elsewhere.

Reply:
How retcodes are related to progress service? I think they are two independent things.

Reply:
Progress returns XML containing state and code. I wasn't sure if retcodes are always int's (as for eg. they could be HEX or other form), but now I think it would be safe to assume it will always be an int and worry in the future if it isin't.